src/Microsoft.DotNet.XHarness.iOS.Shared/AppBundleInformationParser.cs

```diff
@@ -56,14 +57,17 @@ public AppBundleInformation ParseFromProject(string projectFilePath, TestTarget
                 csproj.GetOutputPath(platform, buildConfiguration).Replace('\\', Path.DirectorySeparatorChar),
                 appName + (extension != null ? ".appex" : ".app"));
 
+            string arch = csproj.GetMtouchArch(platform, buildConfiguration);
+            bool supports32 = arch.Contains("ARMv7") || arch.Contains("i386");
```

Comment:
Do you want `StringComparison.InvariantCultureIgnoreCase` or these are always capitalized? Just asking because the other Arm const is in a different case

Reply:
@premun better to ignore the case, good point.